### PR TITLE
:art: Drop scripts and rework ipxe

### PR DESCRIPTION
**Pull request by Itxaka**
Use the bundled template on the next osbuilder release to get the ipxe script wiht the proper values and simplify unused stuff in the repo

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by Itxaka** [comment]
blocked until the new osbuilder release bundled with ipxe template

File: Earthfile [hunk 1]
```diff
@@ -660,18 +660,21 @@ netboot:
     ARG VERSION=$(cat VERSION)
     ARG ISO_NAME=${OS_ID}-${VARIANT}-${FLAVOR}-${TARGETARCH}-${MODEL}-${VERSION}
     ARG OSBUILDER_IMAGE
+    ARG RELEASE_URL=https://github.com/kairos-io/kairos/releases/download
+
+    ENV RELEASE_URL=$RELEASE_URL
```
Comment (jimmykarily) [inline Earthfile:665]: aren't `ARG`s automatically `ENV`s? I think there are several target where we don't do this kind of assignment and it works.

**Review by mauromorales — APPROVED** [review]
lgtm

File: Earthfile (hunk 1 above)
Comment (Itxaka) [reply Earthfile:665]: mmmh, no idea, you might be right

Comment (Itxaka) [reply Earthfile:665]: yepp, seems correct, let me drop them!

**Comment by Itxaka** [comment]
Netboot tests passed correctly, extra ENV dropped.

**Review by mauromorales — APPROVED** [review]

**Pull request merged**